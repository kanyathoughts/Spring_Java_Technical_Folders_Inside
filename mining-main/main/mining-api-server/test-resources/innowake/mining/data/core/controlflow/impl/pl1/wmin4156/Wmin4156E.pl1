 SIMPLE: PROCEDURE OPTIONS(MAIN);
    SELECT (RESULT); 
    WHEN (RESULT)
     PUT SKIP LIST(' MATCH FOUND');
    OTHERWISE PUT SKIP LIST(' NO MATCH FOUND');
    END;
 END SIMPLE;