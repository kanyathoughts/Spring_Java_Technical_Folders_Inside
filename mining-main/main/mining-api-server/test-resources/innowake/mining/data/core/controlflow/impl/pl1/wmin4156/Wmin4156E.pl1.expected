 #2 ProcedureBlock, FROM={TestEntryPoint}, TO={#6}
  #6 ProcedureStatement, FROM={#2}, TO={#15}
  #15 SelectGroup, Label: SELECT (RESULT); WHEN (RESULT) PUT SKIP LIST(' MATCH FOUND'); OTHERWISE PUT SKIP LIST(' NO MATCH FOUND'); END;, FROM={#6}, TO={#33(RESULT), #50(OTHERWISE)}
   #16 SelectStatement
   #25 WhenStatement
    #33 PutStatement, FROM={#15(RESULT)}, TO={#65}
   #48 OtherwiseStatement
    #50 PutStatement, FROM={#15(OTHERWISE)}, TO={#65}
   #65 EndStatement, FROM={#33, #50}, TO={#68}
  #68 EndStatement, FROM={#65}, TO={TestHaltPoint}
